Lombiq.Tests.UI.Samples/Tests/MonkeyTests.cs

```diff
@@ -109,7 +109,7 @@ public MonkeyTests(ITestOutputHelper testOutputHelper)
         };
 
     private static bool IsValidAdminBrowserLogMessage(LogEntry message) =>
```

Comment:
Using similar naming eg `IsValidAdminBrowserLogEntry` would better, if we change `message` to `logEntry`

Reply:
Yup, just didn't notice it.